### PR TITLE
[CL-58] Setup Prisma

### PR DESCRIPTION
**Pull request by guarde**

### CODE REVIEW

File: docker-compose.local.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+  postgres:
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=root
```
Comment (stevelw) [inline docker-compose.local.yaml:5]: Are these actual credentials?

File: prisma/migrations/20241118164752_/migration.sql
Comment (stevelw) [inline prisma/migrations/20241118164752_/migration.sql:1]: Is this file auto-generated, and so should be .gitignored?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ☁ Clowder Backend
+
+## Prerequisites  
+This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance
+
+## Setup
+### Setting up the database
+
+1. Ensure that Docker is started. This is down to how you have installed Docker. You can test Docker is online by running `docker info`
+2. Run `./setup-db.sh local up -d`.
```
Comment (stevelw) [inline README.md]: 1. This should be added as a script in package.json
2. When I run this command I get the error:

```
✘ postgres Error failed to resolve reference "docker.io/library/postgres:14": failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://auth.docker.io/token?s...                 1.1s 
Error response from daemon: failed to resolve reference "docker.io/library/postgres:14": failed to authorize: failed to fetch oauth token: unexpected status from GET request to https://auth.docker.io/token?scope=repository%3Alibrary%2Fpostgres%3Apull&service=registry.docker.io: 401 Unauthorized
```

**Review by stevelw — DISMISSED** [review]

File: docker-compose.local.yaml (hunk 1 above)
Comment (guarde) [reply docker-compose.local.yaml:5]: They are dev environment credentials. They aren't critical to be exposed - They are there to set up the Psql instance on Docker with a consistent username:password

File: prisma/migrations/20241118164752_/migration.sql
Comment (guarde) [reply prisma/migrations/20241118164752_/migration.sql:1]: Auto-generated but Prisma uses it in the migration lock system, so needs to be pushed

File: README.md (hunk 2 above)
Comment (guarde) [reply README.md]: That error seems to be an issue with your local Docker configuration, nothing I can fix in this codebase. What script would you want to put in the package.json?

**Review by mizamidou — APPROVED** [review]

**Review by mizamidou — APPROVED** [review]

**Review by stevelw — APPROVED** [review]

**Pull request merged**